.ls-lint.yml

```diff
@@ -13,4 +13,5 @@ ls:
 
 ignore: 
   - .git 
-  - node_modules
\ No newline at end of file
+  - node_modules
+  - packages/init/node_modules
```

Comment:
Ah, yes, it seems there is no glob support at the moment in **ls-lint**, will need to [wait for v2](https://github.com/loeffel-io/ls-lint/issues/35#issuecomment-737986143) it seems